app/views/candidate_interface/unsubmitted_application_form/show.html.erb

```diff
@@ -45,21 +45,47 @@
         <p class="govuk-body">It takes 8 days to get a reference on average.</p>
       <% end %>
 
-      <ol class="app-task-list">
-        <li class="app-task-list__item">
-          <%= render(TaskListItemComponent.new(
-            text: @application_form_presenter.references_link_text,
-            completed: @application_form_presenter.enough_references_provided?,
-            path: @application_form_presenter.references_path,
-            custom_status: @application_form_presenter.references_in_progress? ? 'In progress' : nil,
-          )) %>
-          <% if @application_form_presenter.references.present? %>
-            <%= render(TaskListItemReferencesComponent.new(
-              references: @application_form_presenter.references,
+      <% if FeatureFlag.active?(:reference_selection) %>
+
+        <ol class="app-task-list">
+          <li class="app-task-list__item">
+            <% if @application_form_presenter.references.present? %>
+              <%= render(TaskListItemReferencesComponent.new(
+                references: @application_form_presenter.references,
+                text: 'Review your references',
+                path: @application_form_presenter.references_path,
+              )) %>
+            <% end %>
+          </li>
+
+          <li class="app-task-list__item">
+            <%= render(TaskListItemComponent.new(
+              text: 'Selected references',
```

Comment:
Could make this an entry in this locale file https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/config/locales/en.yml#L233